.github/workflows/ci.yml

```diff
@@ -59,15 +59,19 @@ jobs:
     timeout-minutes: 5
     env:
       # renovate: datasource=github-releases depName=rhysd/actionlint
-      ACTIONLINT_VERSION: '1.7.4'
+      ACTIONLINT_VERSION: "1.7.4"
```

Comment:
This and other changes in this file look like your editor auto-formats the file on save but I'd like to keep this PR to the required changes only, if possible.